src/menu.cc

```diff
@@ -7,7 +7,7 @@
 Menu::Menu() {
   auto accels=Config::get().menu.keys;
   for(auto &accel: accels) {
-#ifdef JUCI_UBUNTU_BUGGED_MENU
+#ifdef JUCI_UBUNTU
     size_t pos=0;
```

Comment:
Ubuntu's gtk menu-proxy is bugged (at least for Ubuntu 14 and 15, not tested in Ubuntu 16), so this is needed for the key bindings to work. If Ubuntu was not bugged, its key bindings would be set in the Menu::set_keys function. 

They might have fixed it however and backported the fix to 14 and 15. Have not tested this in a while.